kolibri/plugins/audio_mp3_render/assets/src/vue/index.vue

```diff
@@ -45,7 +45,8 @@
 
 <script>
 
-  require('html5media/dist/api/1.1.8/html5media');
+  require('imports?this=>window!html5media/dist/api/1.1.8/html5media');
```

Comment:
if you take a look at my first commit, I deleted the `webpack.config.js` file because for some reason, webpack does not pick up that config. So this inline loader is a workaround for the preceding issue.